examples/ByUrl.purs

```diff
@@ -84,4 +85,5 @@ examples = Map.fromFoldable
   , "ReferenceInputs" /\ ReferenceInputsV2.contract
   , "ReferenceInputsAndScripts" /\ ReferenceInputsAndScriptsV2.contract
   , "Utxos" /\ Utxos.contract
+  -- , "ApplyArgs" /\ ApplyArgs.contract
```

Comment:
What's the plan for this test? This would force us to keep the ctl-server in the code and in the runtime for tests (this include the suit added to Unit tests).

Reply:
I'd like to get @klntsky  opinion here. Idk

But from the runtime i already removed the ctl server. This one test spawns the server for himself. We do keep the haskell server code tho.

Reply:
https://github.com/Plutonomicon/cardano-transaction-lib/pull/1325#discussion_r1048537492

We can remove the code after that

Reply:
Yep